### PR TITLE
Added option to switch camera view

### PR DESCRIPTION
**Pull request by fillnye**

### CODE REVIEW

File: controls_pyqt/gui.py [hunk 1]
```diff
@@ -215,6 +216,21 @@ def __initialize_layout(self):
         # self.layout.setColumnMinimumWidth(3, 400)
         self.layout.setColumnMinimumWidth(4, 600)
         self.setLayout(self.layout)
+    
+     def __switch_camera_layout(self):
+        if self.current_camera_layoutt == 0:
```
Comment (lutetjeff) [inline controls_pyqt/gui.py]: typo here

File: controls_pyqt/gui.py [hunk 2]
```diff
@@ -215,6 +216,21 @@ def __initialize_layout(self):
...
+            self.camera_view2.hide()
+            self.current_camera_layout = 1
```
Comment (lutetjeff) [inline controls_pyqt/gui.py:223]: consider removing the `return` statements and place the constant setting of `self.current_camera_layout` with a `self.current_camera_layout += 1` and `self.current_camera_layout %= CAMERA_COUNT` or something similar

File: controls_pyqt/gui.py [hunk 3]
```diff
@@ -254,6 +270,8 @@ def keyPressEvent(self, key_event: QKeyEvent):
             self.on_trigger("e", True)
         if key_event.key() == Qt.Key_Return:
             self.starttime = time.time()
+	if key_event.key() == Qt.Key_Slash
```
Comment (lutetjeff) [inline controls_pyqt/gui.py:273]: indentation problem here

File: controls_pyqt/gui.py [hunk 4]
```diff
@@ -38,6 +38,7 @@ def __init__(self, mcu_object: MCUInterface, comms: Communications, app: QT.QApp
         self.thruster_speed: List[int] = [0] * self.NUM_THRUSTERS
         self.servo_speed: int = 0
         (self.X_INDEX, self.Y_INDEX, self.Z_INDEX) = (0, 1, 2)
+	self.current_camera_layout = 0
```
Comment (lutetjeff) [inline controls_pyqt/gui.py]: indentation problem here.

**Review by lutetjeff — CHANGES_REQUESTED** [review]
some issues to fix.
consider renaming this to "updated camera view system" and also implement the third camera and its systems necessary here.

**Comment by fillnye** [comment]
Using a different codebase for this enhancement.

**Pull request closed without merging**